pkg/evpn/vrf.go

```diff
@@ -18,7 +18,9 @@ import (
 	"github.com/google/uuid"
 	"github.com/vishvananda/netlink"
 
+	badger "github.com/dgraph-io/badger/v4"
```

Comment:
try to avoid importing `badger` directly here 
you using it mostly for error checking, you should translate badger errors to infradb errors and hide the fact we actually using badger in this file... only main should know about implementation details like this

Reply:
I will fix that